app/static/graph.js

```diff
@@ -0,0 +1,47 @@
+function makeGraph(events) {
```

Comment:
Which graph? Please change the names in the code to be more indicative :)

Reply:
done